### DISCUSSION TITLE
Why I can not find our new added device?

### DISCUSSION THREAD

**Discussion by baggiowu** [Koenkk/zigbee2mqtt#23407] [Devices]
I have added our devices, such as NAS-WV03B2.
Why I can not find our new added device in the site of  https://www.zigbee2mqtt.io/supported-devices/.
I see my Pull request is merged and a new version has been released.

**Comment by Koenkk** [comment]
New devices are included in the dev branch initially and in the next z2m release (every 1st of the month). After the z2m release the docs will be updated.
